test/parallel/test-http2-too-many-streams.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const Countdown = require('../common/countdown');
+const http2 = require('http2');
+
+const server = http2.createServer({ settings: { maxConcurrentStreams: 1 } });
+
+server.on('stream', common.mustCall((stream) => {
+  stream.respond();
+  setImmediate(() => stream.end('ok'));
+}, 2));
+
+server.listen(0, common.mustCall(() => {
+  const client = http2.connect(`http://localhost:${server.address().port}`);
+
+  client.on('sessionError', console.log);
```

Comment:
Are these logs needed? To be explicit maybe wrap them in `common.must(Not)Call`.

Reply:
oh, no, they're not. just forgot to remove them